src/main/java/ro/unibuc/hello/service/DoneeService.java

```diff
@@ -0,0 +1,42 @@
+package ro.unibuc.hello.service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.bson.types.ObjectId;
+import org.springframework.beans.BeanUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import ro.unibuc.hello.data.CharityEventRepository;
+import ro.unibuc.hello.data.DoneeEntity;
+import ro.unibuc.hello.dto.DoneeDTO;
+
+@Component
```

Comment:
```suggestion
@Service
```

They do the exact same thing (facilitate dependency injection), but is more specific. Don't forget to also import it, if you make this change :D.